src/CommunityVoices/App/Website/Presentation/Module/Image.xslt

```diff
@@ -5,6 +5,8 @@
   <xsl:output method="html" indent="yes" omit-xml-declaration="yes" />
 
   <xsl:variable name="isAdmin" select="package/identity/user/role = 'administrator'"/>
+  <xsl:variable name="isManager" select="$isAdmin or package/identity/user/role = 'manager'"/>
```

Comment:
Reasoning?

Reply:
Because certain actions like creating a new slide should also be available to managers, not just admins.

Reply:
Yep, didn't realize this was used elsewhere in the file.  My bad.